draft-foudil-securitytxt.md

```diff
@@ -317,6 +322,8 @@ directive.
 As stated in {{encryption}} and {{signature}}, both encryption keys
 and external signature files SHOULD be loaded over HTTPS.
 
+Web services MUST reserve the security.txt namespace to ensure no third-party can create a page with the "security.txt" name.
```

Comment:
Can we explain what this means? "Web services" has multiple meanings. Are we talking about a multi user host like Wordpress.com or Blogger?

Reply:
@nightwatchcyber, is there an RFC that we could reference here in order to define https://en.wikipedia.org/wiki/Web_service?

Reply:
Can we first figure out what we mean by "web service"? In my mind I was thinking of a domain name or IP addresses providing connections via HTTP protocol on port 80, but not necessarily machine to machine only. What is your thought?